lib/video/screenRecording/desktop/firefoxrecorder.js

```diff
@@ -0,0 +1,161 @@
+'use strict';
+
+const log = require('intel').getLogger('browsertime.video');
+const os = require('os');
+const util = require('util');
+const fs = require('fs');
+const path = require('path');
+const delay = ms => new Promise(res => setTimeout(res, ms));
+const execa = require('execa');
+const del = require('del');
+
+const unlink = util.promisify(fs.unlink);
+
+async function createTempFile() {
+  const mkdtemp = util.promisify(fs.mkdtemp);
+  const dir = await mkdtemp(path.join(os.tmpdir(), 'browsertime-'));
+  return path.join(dir, 'winrecorder.mp4');
+}
+
+async function enableWindowRecorder(enable, driver) {
+  if (enable) {
+    enable = '1';
+  } else {
+    enable = '0';
+  }
+  const oldContext = driver.getContext();
+  await driver.setContext('chrome');
+  const script = 'windowUtils.setCompositionRecording(' + enable + ');';
+  await driver.executeScript(script);
+  return driver.setContext(oldContext);
+}
+
+function findRecordingDirectory() {
+  let closest_mtime = 0;
+  let directory = undefined;
+
+  fs.readdirSync('.').forEach(file => {
```

Comment:
What are you trying to find?  This doesn't find the "smallest greater than", it finds the "greatest greater than".  Maybe collect all `(mtime, file)`, sort, filter, take the head?  Can we not just save something that will allow us to determine this file number?  Maybe we need to make Firefox tell us where it's writing the files?